dio/lib/src/dio/dio_for_browser.dart

```diff
@@ -29,9 +27,31 @@ class DioForBrowser with DioMixin implements Dio {
     String lengthHeader = Headers.contentLengthHeader,
     Object? data,
     Options? options,
-  }) {
-    throw UnsupportedError(
-      'The download method is not available in the Web environment.',
+  }) async {
+    options ??= DioMixin.checkOptions('GET', options);
+
+    options = options.copyWith(responseType: ResponseType.bytes);
```

Comment:
So the download would wait until all bytes got saved, which will likely cause high memory usage and long waiting, right?

Reply:
Yes, it should wait until it is finished, but it can be checked with onReceiveProgress

Reply:
@AlexV525 

I am trying to do this with stream, but the data is not returned through stream! Is there a problem with stream on the web side?

Reply:
changed